docs/src/concepts.md

```diff
@@ -1,23 +1,122 @@
 # Concepts
 
-<!-- ## BossDB Data -->
+Volumetric calcium imaging is a relatively new technique in neuroscience but advances in
+computational capabilities have facilitated a rapid rise in its development and
+use. As more labs use volumetric calcium imaging to
+understand the brain, standardization of annotations and raw data storage have
+emerged as some of the greatest challenges in the field. 
+
+This element consists of a pipeline to perform volume matching across sessions
+and cell segmentation using `cellpose` for volumetric microscopic calcium imaging data, and an
+interface to easily upload the results and raw data to the Brain Observatory
+Storage Service and Database (BossDB).
```

Comment:
I don't think that this is limited to calcium imaging data.  Rather any volumetric fluorescent microscopy data would work with cellpose.

Reply:
Yep, this is true and updated in the most recent commit